### ISSUE TITLE
Missing scroll feature

### ISSUE THREAD

**Issue by Sentinel8000** [adafruit/Adafruit_SSD1305#27]
I have ordered a 2.23inch OLED HAT

https://www.waveshare.com/wiki/2.23inch_OLED_HAT

This working with SS1305 library. I want use large fonts, but so i need a scrolling feature from this library - without this library/display not usable with larger fonts. Manual scrolling programing is not all case usable- 

Also this feature included in SSD1306 library and realy missing from here....

**Comment by ladyada** [comment]
yep that would be great! have you asked waveshare to write that code and submit it?

**Comment by Sentinel8000** [comment]
Sorry, i dont know who is responsible or can help.... - because im using this library as ,,n00b" i have thinked, the library creator can help. However thanks for the nice response:)

**Comment by ladyada** [comment]
hiya you can contact them at https://support.waveshare.com/hc/en-us/requests/new did you try that?

**Comment by Sentinel8000** [comment]
Thanks, after your first answer, i have sendet this request to the support. 

However im not sure, this feature will be created, because im not sure im so lucky these people will be develop just because me this feature:)

So lookslike i have a not cheap 2.23" OLED display what im unable to use for my hobby project - display longer texts :D

**Comment by Sentinel8000** [comment]
I got answer, but lookslike this is death theme:

,,Hi，

Sorry for replying late,

The datasheet of SSD1305 indicates that scrolling can use internal registers to achieve. I‘m sorry that we don't have the demo for this function now, but we will try to update it as soon as possible."

![unnamed](https://user-images.githubusercontent.com/68690602/154228789-f15d3b84-fd8d-4ced-acd8-1ca00c43d045.png)
